AEXS/aex-5.md

```diff
@@ -6,7 +6,7 @@ Title: Inter-Wallet Communication
 Author: Shubhendu Shekhar <@shekhar-shubhendu>
 License: BSD-3-Clause
 Discussions-To: https://forum.aeternity.com/t/aex-5-inter-wallet-communication/3371
-Status: Draft
+Status: Last call (2019-05-29 to 2019-06-04)
```

Comment:
Shouldn't it be changed to "Review" firstly?

Reply:
we've been through various discussions and I guess we can say it has been through review and lot of scrutiny. moving to review is only for symbolic reasons then I think we should skip it and save some time and money.

Reply:
> Each proposal must start in the Draft state and will move through the phases in accordance to the criteria defined in this document.

https://github.com/aeternity/AEXs/blob/master/AEXS/aex-1.md#preamble

Reply:
@davidyuk I think just the representation of these stages was missing for this one. `Active Review` was done in various discussions and calls, but was forgotten to update in the repository.